CompuCell3D/CMakeLists.txt

```diff
@@ -45,7 +45,12 @@ OPTION(FLEX_PYTHON_SPECS "Allow Flexible Python Path specification" OFF)
 
 
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development NumPy)
-
+if(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+    get_filename_component(PYTHON_BASE_DIR ${Python3_EXECUTABLE} DIRECTORY)
+else(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
+    get_filename_component(PYTHON_BASE_DIR ${Python3_EXECUTABLE} DIRECTORY)
+    get_filename_component(PYTHON_BASE_DIR ${PYTHON_BASE_DIR} DIRECTORY)
+endif()
```

Comment:
Recommended code:
```
if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
    get_filename_component(PYTHON_BASE_DIR "${Python3_EXECUTABLE}" DIRECTORY)
else()
    get_filename_component(PYTHON_BASE_DIR "${Python3_EXECUTABLE}" DIRECTORY)
    get_filename_component(PYTHON_BASE_DIR "${PYTHON_BASE_DIR}" DIRECTORY)
endif()
```
1. this eliminates the redundant condition
2. adding quotes ensures that directories containing spaces can be handled

Reply:
Fixed

Reply:
I see that the redundant condition was removed, but I do not see the quotes. Did you have a strong preference for one way over another?

Reply:
this is resolved although I don't know why GitHub does not update it here:
https://github.com/CompuCell3D/CompuCell3D/blob/feature/ocl_fixes_new/CompuCell3D/CMakeLists.txt

Reply:
Nice, thank you for the new commit (a7fc6fafb94cb7cf251f8825fd48e87cae07e6b5). I still see a few places in this file where we might need quotes to handle directories, but I do not wish to be too pedantic. Your choice to fix them. 
PR approved.